### ISSUE TITLE
Bank Trucks (AI Health)

### ISSUE THREAD

**Issue by NerxaOG** [FrenzyRoleplay/server-issues#20]
The AI is way too powerful. The point of doing the job is to open a gateway for a potential action-packed scene in which cops and crim can have a shootout. Many people have agreed. Most bank trucks end up with 1 or 2 down before the back can even start being hit. It takes nearly 2 full magazines from vintage to even drop 1 guard and they all have to be perfect headshots, even then they sometimes are still standing.

Dropping the AI health IMO would better benefit the outcome of the scene. By allowing criminals to save their resources for a gunfight and allowing RP to build would be far more beneficial than leaving it to where it will mostly be a 3v6 or 2v6.

**Comment by badcodestv** [comment]
Not sure if I whole heartedly agree here. Bank Trucks are suppose to be hard to hit. If it was easy, then everyone would be able to hit them. In my opinion, I think the armed guards health is perfectly fine. Let's see what the rest of the community thinks?

**Comment by Vteckia** [comment]
I agree with codes here. I think the bank truck should be a pretty hard to hit over another robberies. If getting downed by AI is an issue maybe better planning or supplies to take on the job fully should accounted for

**Comment by ProximateTv** [comment]
I think with Aps or class 2's the challenge is appropriate but the vintage definitely took close to 4 clips and the guards weapons downed me in about 5 seconds(Perhaps it's because I'm big dumb and didn't take cover and somehow drew the aggro of all 3 guards). If we could use automatics and switch to non autos for any ensuing gunfight without fear of the PD AR coming out I think it is appropriate risk vs reward personally, especially considering the low cost to pull off the job.

**Comment by Vteckia** [comment]
What about the possibility of having the guard either having class 1s or class 2 having it at a random?

**Comment by badcodestv** [comment]
> If we could use automatics and switch to non autos for any ensuing gunfight without fear of the PD AR coming out

I've seen this happen quite a few times and it works out just fine. PD is also aware that the initial shots fired will be automatic since they know the armed guards are carrying autos so once a "gunfight" has been initiated after the fact, that's when PD will match force depending on what type of weapon is fired after the fact.

**Comment by badcodestv** [comment]
> What about the possibility of having the guard either having class 1s or class 2 having it at a random?

This wouldn't be hard to implement.

**Comment by ProximateTv** [comment]
> I've seen this happen quite a few times and it works out just fine. PD is also aware that the initial shots fired will be automatic since they know the armed guards are carrying autos so once a "gunfight" has been initiated after the fact, that's when PD will match force depending on what type of weapon is fired after the fact.

Sick... I think that makes it much more practical

**Comment by Swiaa** [comment]
In my experience with the bank trucks I know the guards use class 2 weapons and are heavily armored, so my crew prepares for that by matching the guards firepower and setting up positions around the truck to angle against every guard. Once we've taken out all the guards we switch to our class 1 weapons against the PD, unless we all decide we want it to be a class 2 fight. I've personally never had a problem with the bank trucks firepower or armor. It's all tactics :)

**Comment by Swiaa** [comment]
So yeah, I feel like making the bank truck guards weaker wouldn't be as tactic or fun.

**Comment by NerxaOG** [comment]
I suggest making them weaker instead of sacrificing class 2 for the sake of a scene when we want to stick with class 1 against PD. I think it would build far better RP for both parties if we dropped the AI health and allowed 8 Cops to respond instead of 6. I understand the mindset of having them hard, but when you have to mag dump an AI with 3 magazines then prepare for a police shootout it takes away imo the RP portion and makes it feel more like CoD where the point is kill anything that moves. By the time guards are dropped and people are still grabbing the loot. Cops are already there shooting. Instead, it could be slower at first and there could be small banter or an actual interaction before it turns into nuketown.

**Comment by DeviLzFury** [comment]
I have done 1 Bank Truck and did it with 2 people one had a combat another had a vintage and we still had enough time to hit the truck and dipped without firing a shot at PD. This  was a test run why we didn't stick around for action with PD but to the point of this post AI health is fine IMO if 2 people with class 1s can do it and getaway clean then a team of 4 can easily hit it and still have RP with PD.  If you are loosing 1 or 2 of your 4 man team its not AI health that's the issue.

**Comment by ProximateTv** [comment]
> If you are loosing 1 or 2 of your 4 man team its not AI health that's the issue.

I'll be the first to admit that me being downed was just shitty luck or poor planning on my part for catching aggro from all three guards.
